platforms/qhaikuplatform/haiku/qhaikuwindow.cpp

```diff
@@ -915,7 +916,8 @@ void QHaikuWindow::swapBuffers()
 
 BRegion QHaikuWindow::getClippingRegion()
 {
-	BRegion region(BRect(0, 0, window()->width(), window()->height()));
+	const QSize nativeSize = QHighDpi::toNativePixels(window()->size(), window());
```

Comment:
```suggestion
	const QSize nativeSize = windowGeometry().size();
```

See QPlatformWindow::windowGeometry()

Reply:
Aha, indeed.